apps/parsley/src/constants/errors.ts

```diff
@@ -6,4 +6,4 @@ export const LOG_LINE_TOO_LARGE_WARNING = `Parsley was unable to process one or
 )}. They have been trimmed to fit the limit.`;
 
 export const LOG_FILE_DOWNLOAD_TOO_LARGE_WARNING =
-  "Parsley was only able to partially download this log. Use the Evergreen CLI command in the details menu to download the log onto your machine.";
+  "Parsley was only able to partially download this log. You can use a command line tool to download the file from the raw URL found in the 'Details' menu.";
```

Comment:
Should we say 
```suggestion
  "Parsley was only able to partially download this log. You can use `wget` or `curl`  to download the file from the raw URL found in the 'Details' menu.";
```
to eliminate any confusion on what command line tool they should use?